### PR TITLE
adding container registry replication module

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Adding container registry replication module

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
azure_rm_containeregistryreplication

##### ANSIBLE VERSION
2.7

### CODE REVIEW

**Comment by Fred-sun** [comment]
@zikalino

**Comment by Fred-sun** [comment]
@zikalino

**Comment by Fred-sun** [comment]
@zikalino

**Comment by Fred-sun** [comment]
@yuwzho @yungezz need review

**Comment by Fred-sun** [comment]
@yungezz @yuwzho Please help review.

File: lib/ansible/modules/cloud/azure/azure_rm_containerregistryreplication.py [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_containerregistryreplication
+version_added: "2.5"
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_containerregistryreplication.py:19]: seems this pr out of date. @zikalino pls update when you're available

**Comment by Fred-sun** [comment]
@zikalino

**Comment by Fred-sun** [comment]
@zikalino

**Comment by Fred-sun** [comment]
@zikalino Please help take a look this PR and push for merge, when you're available! Thanks!

**Comment by Fred-sun** [comment]
@zikalino

**Comment by Fred-sun** [comment]
@zikalino need_update

**Comment by Fred-sun** [comment]
kindly ping

**Comment by Fred-sun** [comment]
@zikalino please update this PR according by the comments? Thanks!

**Comment by Fred-sun** [comment]
@zikalino need you update,  out of date! Thanks!

**Comment by Fred-sun** [comment]
@zikalino

**Comment by Fred-sun** [comment]
@zikalino Please resolve the shippable error and changed according by comments! Thank you very much!

**Comment by Fred-sun** [comment]
@zikalino

**Comment by Fred-sun** [comment]
@zikalino Need you to update! Please update when you're free! Thank you very much!

**Comment by Fred-sun** [comment]
@MyronFanQiu The azure_rm_containerregistryreplication module has been added to ansible 2.9.0devel. Can we close the PR? Please confirm! Thank you very much!

**Pull request closed without merging**